File: study-2nd/level1/yongseong/Solution02.js
function solution(x, n) {
    var array = [];
    const plusNumber = x;
    // n만큼 반복합니다.
    for ( var i = 0; i < n; i++ ) {
        // x를 배열에 삽입합니다.
        array.push(x);
        // x를 증가시킵니다.
        x += plusNumber;
    }
    // 배열을 리턴합니다.
    return array;
}

File: study-1st/team1/BF/jiwon/Solution02.py
from itertools import permutations

def primen(num):
    if num > 1:
        for i in range(2, num//2 + 1):
            if (num/i) == (num//i):
                return False
            else:
                return True
    else:
        return False

def solution(numbers):
    p = list()
    s = list()
    count = 0
    for i in range(len(numbers)):
        n = list(permutations(list(numbers), i+1))
        p.extend(n)
    for i in range(len(p)):
        s.append("".join(p[i]))
    s = list(set(list(map(int, s))))
    for num in s:
        if primen(num):
            count += 1
    return count

numbers = "17"
print(solution(numbers))